### ISSUE TITLE
object-curly-newline minProperties documentation clarification

### ISSUE THREAD

**Issue by solmsted** [eslint/eslint#8874]
**Tell us about your environment**

* **ESLint Version:** 4.1.1
* **Node Version:** 8.0.0
* **npm Version:** 5.0.4

**What parser (default, Babel-ESLint, etc.) are you using?** default

The documentation for `object-curly-newline`'s `minProperties` option states

```
"minProperties" requires line breaks if the number of properties is more than the given integer.
```

In practice I have found that it requires line breaks when the number of properties is not simply `more than` the given integer but *greater than or equal to* the given integer.

Here is an example:

```JavaScript
/* eslint object-curly-newline: [2, {minProperties: 1}] */
let obj = {a: 1};
```

This triggers errors:
* ESLint Error object-curly-newline Expected a line break after this opening brace.
* ESLint Error object-curly-newline Expected a line break before this closing brace.

This object has one property and `minProperties` is set to one.  One is not "more than" one.  So I believe the documentation is incorrect.

**Comment by not-an-aardvark** [comment]
Thanks for reporting, this looks like a typo in the docs. Would you like to submit a PR?
